app/Http/Controllers/CompetitionsController.php

```diff
@@ -59,7 +60,13 @@ public function store(CompetitionRequest $request)
      */
     public function show(Competition $competition)
     {
-        return view('competitions.show', compact('competition'));
+        $bracket = Competition::getBracket($competition->id);
```

Comment:
This isn't implement yet right, cause we don't have Brackets yet?

Would this be plural though, since multiple brackets could apply to the `$competition->id`?